styles/base.scss

```diff
@@ -0,0 +1,15 @@
+html {
+	height: 100%;
+	font-size: 16px;
+	font-family: 'Montserrat', sans-serif;
+}
+
+body {
+	height: 100%;
+}
+
+h1 {
+	font-size: 38px;
+	text-transform: uppercase;
+	color: white;
```

Comment:
let's create a separate file with some variables and use them in other styles. Check all your files.
```
$color-white: #ffffff;
$color-gray: ...
...
$font-primary: "Montserrat", sans-serif;
```

Reply:
we have discussed and decided to use css variables everywhere